### PR TITLE
Task #PS-1795 : Board exam enrolment UI Implementation

### PR DESCRIPTION
**Pull request by shreyas1434shinde**

### CODE REVIEW

File: public/locales/en/common.json [hunk 1]
```diff
@@ -569,6 +569,7 @@
     "EXAM_FEES_PAID":"  Exam Fees Paid? (optional)",
     "SAVE_AND_NEXT":"Save and Next",
     "TO_SAVE_YOUR_PROGRESS":"“Save & Next” to save your progress. You can come back and change it anytime.",
-    "MANDATORY":"Mandatory for NIOS only. For other Boards, this step is automatically marked completed. Please click on Save to submit the data."
+    "MANDATORY":"Mandatory for NIOS only. For other Boards, this step is automatically marked completed. Please click on Save to submit the data.",
```
Comment (itsvick) [inline public/locales/en/common.json:572]: @shreyas1434shinde NIOS should be configurable

Comment (shreyas1434shinde) [reply public/locales/en/common.json:572]: @itsvick it's under box msg which is already rendering on condition

File: app.config.ts [hunk 2]
```diff
@@ -82,3 +82,6 @@ export const RequisiteType = {
   PRE_REQUISITES: 'prerequisite',
   POST_REQUISITES: 'postrequisite',
 };
+export enum BoardNios {
```
Comment (itsvick) [inline app.config.ts:85]: @shreyas1434shinde keep varaible as export const 
FeesStepBoards: ["NIOS"]

Comment (shreyas1434shinde) [reply app.config.ts]: @itsvick resolved

**Pull request merged**